package.json

```diff
@@ -108,7 +108,7 @@
     "use-cjs": "echo '{\n  \"extends\": \"./tsconfig/cjs.json\"\n}' > tsconfig.json",
     "use-mjs": "echo '{\n  \"extends\": \"./tsconfig/mjs.json\"\n}' > tsconfig.json",
     "=============================================================================== aliases": "",
-    "test": "cd tests/; pnpm jest",
+    "test": "cp package.cjs.json cjs/package.json; pnpm jest",
```

Comment:
The `cjs` directory should already have `package.json` file with the `imports` map.  It is copied to that directory from the `components/bin` directory by the last command on the line

https://github.com/mathjax/MathJax-src/blob/2185f0b7e36436cb8b52a7f361fcaf0582d40684/package.json#L93

So there is no need for the `package.cjs.json` file, or the `cp` command here.  You can call

``` shell
pnpm -s copy:pkg cjs
```

if you really want to make sure, but building the `cjs` directory should create the needed `package.json` file.

Reply:
Done.